docs/Makefile

```diff
@@ -1,5 +1,9 @@
 # Minimal makefile for Sphinx documentation
 #
+# copy current tutorials
+copy:
+	cp -r ../tutorial/* source/tutorial/
```

Comment:
would you mind changing `tutorial` to `tutorials` under the root directory and source/tutorials directory?